src/app/common/components/MevRelaysBadge/MevRelaysBadge.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { getImage } from '~lib/utils/filePath';
+import { useStyles } from '~app/common/components/MevRelaysBadge/MevRelaysBadge.styles';
+
+const MevRelaysBadge = ({ label }: { label: string }) => {
+    const classes = useStyles();
+    const logo = label.toLowerCase().includes('bloxroute') ?
+        getImage('mevs/bloxroute.svg') :
```

Comment:
What a strange condition here.
First of all, it's always better to have Record<label, filePath> instead of doing like here.
Secondly, bloxroute image can also be taken as in last operand

Reply:
done